### PR TITLE
{API Management} Fix documentation for `apply-network-updates` command

### PR DESCRIPTION
**Pull request by arhaverly**
**Description**

`apply_network_configuration_updates` function has been renamed to `begin_apply_network_configuration_updates`

The documentation on the `apply-network-updates` command is wrong. This fixes it.

**Testing Guide**

No testing required.

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
API Management

**Review by necusjz — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/apim/_help.py [hunk 1]
```diff
@@ -90,6 +90,15 @@
              --storage-account-key Ay2ZbdxLnD4OJPT29F6jLPkB6KynOzx85YCObhrw==
 """
 
+helps['apim apply-network-updates'] = """
+type: command
+short-summary: Updates the API Management resource running in the virtual network to pick the updated network settings.
+examples:
+  - name: Updates the virtual network settings of the API Management service instance
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/apim/_help.py:97]:
```suggestion
short-summary: Update the API Management resource running in the virtual network to pick the updated network settings.
examples:
  - name: Update the virtual network settings of the API Management service instance
```

File: src/azure-cli/azure/cli/command_modules/apim/custom.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ def apim_restore(client, resource_group_name, name, backup_name, storage_account
 
 
 def apim_apply_network_configuration_updates(client, resource_group_name, name, location=None):
-    """Update the Microsoft.ApiManagement resource running in the Virtual network to pick the updated DNS changes. """
+    """Updates the API Management resource running in the virtual network to pick the updated network settings. """
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/apim/custom.py:182]:
```suggestion
    """Update the API Management resource running in the virtual network to pick the updated network settings. """
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**